spec/graphql/mutations/rock_and_pebble/delete_rock_pebble_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'delete rock and pebble relationship', type: :request do
```

Comment:
Can you remind me what `type: :request` does again? Does that tell RSpec to treat it as an API test?

Reply:
Yes that is correct!